translate/cloud-client/beta_snippets_test.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pytest
+import uuid
+import beta_snippets
+from google.cloud import storage
+
+PROJECT_ID = os.environ['GCLOUD_PROJECT']
+
+
+@pytest.fixture(scope='function')
+def bucket():
+    # Create a temporaty bucket to store annotation output.
+    bucket_name = str(uuid.uuid1())
+    storage_client = storage.Client()
+    bucket = storage_client.create_bucket(bucket_name)
+
+    yield bucket
+
+    # Teardown.
+    bucket.delete(force=True)
+
+
+def test_translate_text(capsys):
+    beta_snippets.translate_text(PROJECT_ID, 'Hello world')
+    out, _ = capsys.readouterr()
+    assert 'Zdravo svet' in out
+
+
+def test_batch_translate_text(capsys, bucket):
+    beta_snippets.batch_translate_text(
+        PROJECT_ID,
+        'gs://cloud-samples-data/translation/text.txt',
+        'gs://{}/translation/BATCH_TRANSLATION_OUTPUT/'.format(bucket.name))
+    out, _ = capsys.readouterr()
+    assert 'Total Characters: 13' in out
+    assert 'Translated Characters: 13' in out
+
+
+def test_detect_language(capsys):
+    beta_snippets.detect_language(PROJECT_ID, 'Hæ sæta')
+    out, _ = capsys.readouterr()
+    assert 'is' in out
+
+
+def test_list_languages(capsys):
+    beta_snippets.list_languages(PROJECT_ID)
+    out, _ = capsys.readouterr()
+    assert 'zh-CN' in out
+
+
+def test_list_languages_with_target(capsys):
+    beta_snippets.list_languages_with_target(PROJECT_ID, 'is')
+    out, _ = capsys.readouterr()
+    assert u'Language Code: sq' in out
+    assert u'Display Name: albanska' in out
+
+
+def test_create_glossary(capsys):
+    beta_snippets.create_glossary(PROJECT_ID, 'glossary')
+    out, _ = capsys.readouterr()
+    assert 'gs://cloud-samples-data/translation/glossary.csv' in out
+
+
+def test_get_glossary(capsys):
+    beta_snippets.get_glossary(PROJECT_ID)
+    out, _ = capsys.readouterr()
+    assert 'gs://cloud-samples-data/translation/glossary.csv' in out
+
+
+def test_list_glossary(capsys):
+    beta_snippets.list_glossaries(PROJECT_ID)
+    out, _ = capsys.readouterr()
+    assert 'gs://cloud-samples-data/translation/glossary.csv' in out
+
+
+def test_translate_text_with_glossary(capsys):
+    beta_snippets.translate_text_with_glossary(PROJECT_ID, 'directions')
+    out, _ = capsys.readouterr()
+    assert 'direcciones' in out
+
+
+def test_delete_glossary(capsys):
+    beta_snippets.delete_glossary(PROJECT_ID)
```

Comment:
This shouldn't work.
Where does the glossary come from?
What's the ID of the glossary?